app/certificates.go

```diff
@@ -0,0 +1,429 @@
+package app
+
+import (
+	"bytes"
+	"crypto/ecdsa"
+	"crypto/elliptic"
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/pem"
+	"fmt"
+	"io/ioutil"
+	"math/big"
+	"time"
+
+	"github.com/go-playground/validator/v10"
+	"github.com/temporalio/tcld/utils"
+	"github.com/urfave/cli/v2"
+)
+
+func generateRandomString(n int) (string, error) {
```

Comment:
can we be consistent with how we generate random strings with what we are doing in the auth package?